pkg/languagedetection/parsing.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package languagedetection
+
+import (
+	"path/filepath"
+	"strings"
+	"unicode"
+)
+
+func isRuneLetterAt(s string, position int) bool {
+	return len(s) > position && unicode.IsLetter(rune(s[position]))
+}
+
+// parseExeStartWithSymbol deals with exe that starts with special chars like "(", "-" or "["
+func parseExeStartWithSymbol(exe string) string {
+	if exe == "" {
+		return exe
+	}
+	// drop the first character
+	result := exe[1:]
+	// if last character is also special character, also drop it
+	if result != "" && !isRuneLetterAt(result, len(result)-1) {
+		result = result[:len(result)-1]
+	}
```

Comment:
it removes numbers as well. is that expected?

Reply:
I think for use case probably no. I will change it so we keep alphanumeric runes.